src/esp/sensor/VisualSensor.h

```diff
@@ -108,9 +110,18 @@ class VisualSensor : public Sensor {
     return false;
   }
 
+  /**
+   * @brief Returns RenderCamera of CameraSensor
+   */
+  gfx::RenderCamera& getRenderCamera() { return renderCamera_; }
+
  protected:
   std::unique_ptr<gfx::RenderTarget> tgt_;
 
+  /** @brief RenderCamera for sensor
+   */
+  gfx::RenderCamera renderCamera_;
```

Comment:
This will need to be a pointer that doesn't have delete called on it by the destructor.  Magnum will cleanup things like gfx::RenderCamera when the scene graph is destroyed so this is currently being double freed (that is what is causing the assert failure on the CI)

Reply:
@vauduong : To address what @erikwijmans pointed out, I would recommend to use a raw pointer (a member variable of the CameraSensor) to point to a renderCamera.
A good reference is here:
https://github.com/facebookresearch/habitat-sim/compare/cubemap#diff-2e8bced9d5e63f6f91584f0cab332f30fe2c372c654ebfbd6ae6485dbe2d6475R116